src/content/guides/how-to-use-a-wallet/index.md

```diff
@@ -0,0 +1,88 @@
+
+1. Open your wallet app.
+2. Click on "Receive" or similarly worded option on the dashboard
+3. Copy your Ethereum address to clipboard.
+4. Provide the sender with your receiving Ethereum address.
+
+If you are trying to send yourself your funds from an exchange it is in principle the same process.
+
+
+## Send cryptocurrency
+
+Would you like to send ETH to another wallet? 
+
+1. Prepare the receiving address and the network of the recipient.
+2. Open your wallet app.
+3. Click on a “Send” button on the dashboard of your wallet or similarly worded alternative.  
+4. Enter the receiving address or scan a QR code with your camera so that you don’t have to write the address manually.
+<br/><br/>
+<img src="./send.png" width="400" alt="Send field for crypto address"/>
+<br/><br/>
+
+4. Many assets, like DAI or USDC, exist on multiple networks. When transferring crypto tokens, make sure that the other side is using the same network as you are. 
+5. Confirm the transaction and have prepared a small amount of ETH to pay for a transaction fee, the exact amount of the fee changes depending on network congestion. 
+6. Once your transaction is processed, the crypto will show up in the recipient’s account. This might take anywhere from few seconds to few minutes.
+
+## Connecting to projects
+
+Your address represents you on all Ethereum projects. You will not need to register individually on any project. Once you have a wallet, you can connect to any Ethereum project without any additional information. No emails or any other personal information are needed.
+
+1. Visit any project’s website.
+2. Click a button “Open the App” in the menu if the main page is a presentation of the project. 
+3. Once you are in the app click on “Connect”
+
+<br/><br/>
+<img src="./connect1.png" width="400" alt="Button allowing user to connect to the website with a wallet"/>
+<br/><br/>
+
+4. Select your wallet from the offered list. If you can't see your wallet, maybe it will be hidden under “WalletConnect” option.
+
+&nbsp;
+<img src="./connect2.png" width="400" alt="Selecting from a list of wallets to connect with"/>
+&nbsp;
+
+5. Confirm a signature request in your wallet to start the connection. It should not cost anything.
+6. Thats it! Start using the app. You can find some interesting projects on our [dApps page](https://ethereum.org/en/dapps/#explore).
+
+<br/><br/>
+
+### FAQ
+
+**If I own an ETH address, do I own the same address on other blockchains?**
+
+- You can use the same address on many popular blockchains (if you have the type of wallet with a recovery phrase). This [list](https://chainlist.org/) will show you which blockchains you can use with the same address. Some blockchains like Bitcoin are completely separate and you will need a different address with a different format. If you have a smart contract wallet you should check its product website for more info on what blockchains are allowed.
```

Comment:
Maybe hinting at EVM compatibility here would be useful?

Something like 'many blockchains are built using the same structure as Ethereum (these are known as EVM compatible blockchain), you can use the same address on many of these blockchains.

Reply:
Yeah, I see some reuse from above... Left a comment up there, but same would apply. A lot of it really does come down to the "structure" of the chain... it's not just that you used a mnemonic, and it's not just new-chain-new-addresss... It comes down to what type of address/signature scheme is used... most chains have their own setup, but those they went with the EVM share that with Ethereum.